packages/react-native-sdk/docusaurus/docs/reactnative/02-tutorials/03-livestream.mdx

```diff
@@ -0,0 +1,448 @@
+:::
+
+#### Android Specific installation
+
+In `android/app/build.gradle` add the following inside the `android` section:
+
+```
+compileOptions {
+	sourceCompatibility JavaVersion.VERSION_1_8
+	targetCompatibility JavaVersion.VERSION_1_8
+}
+```
+
+In `android/gradle.properties` add the following:
+
+```
+android.enableDexingArtifactTransform.desugaring=false
+```
+
+### Step 3 - Broadcast a livestream from your device
+
+The following code shows how to publish from your device's camera.
+Let's open `App.tsx` and replace its contents with the following code:
+
+```tsx title=App.tsx
+import React, { useEffect } from 'react';
+import {
+  StreamCall,
+  StreamVideo,
+  StreamVideoClient,
+  StreamVideoRN,
+  User,
+} from '@stream-io/video-react-native-sdk';
+import { SafeAreaView, Text } from 'react-native';
+
+// for simplicity, we assume that permission was granted through the native settings app
+StreamVideoRN.setPermissions({
+  isCameraPermissionGranted: true,
+  isMicPermissionGranted: true,
+});
+
+const apiKey = 'REPLACE_WITH_API_KEY'; // the API key can be found in the "Credentials" section
+const token = 'REPLACE_WITH_TOKEN'; // the token can be found in the "Credentials" section
+const userId = 'REPLACE_WITH_USER_ID'; // the user id can be found in the "Credentials" section
+const callId = 'REPLACE_WITH_CALL_ID'; // the call id can be found in the "Credentials" section
+
+// initialize the user object
+const user: User = {
+  id: userId,
+  name: 'Santhosh',
+  image: `https://getstream.io/random_png/?id=${userId}&name=Santhosh`,
+};
+
+const myClient = new StreamVideoClient({ apiKey, user, token });
+const myCall = myClient.call('livestream', callId);
+
+export default function App() {
+  useEffect(() => {
+    myCall.join({ create: true }).catch((err) => {
+      console.error('Failed to join the call', err);
+    });
+
+    return () => {
+      myCall.leave().catch((err) => {
+        console.error('Failed to leave the call', err);
+      });
+    };
+  }, []);
+
+  return (
+    <StreamVideo client={myClient} language="en">
+      <StreamCall call={myCall}>
+        <SafeAreaView style={{ flex: 1 }}>
+          <LivestreamUI />
+        </SafeAreaView>
+      </StreamCall>
+    </StreamVideo>
+  );
+}
+
+
+const LivestreamUI = () => (
+  <Text style={{ fontSize: 30, color: 'black' }}>TODO: render video</Text>
+);
+```
+
+If you run the app now, you'll see that the client fails to connect.
+To fix that, we need to provide valid credentials. They shall be set in the following variables:
+
+```ts
+const apiKey = 'REPLACE_WITH_API_KEY';
+const token = 'REPLACE_WITH_TOKEN';
+const userId = 'REPLACE_WITH_USER_ID';
+const callId = 'REPLACE_WITH_CALL_ID';
+```
+
+Replace them now with the appropriate values from the table below:
```

Comment:
Could we set these values before `Broadcast a livestream from your device` in .env file or even just directly in App.tsx or separate file?

So that when user adds the actual code he won't receive error.